go.mod

```diff
@@ -39,44 +39,96 @@ require (
 )
 
 require (
+	github.com/BurntSushi/toml v1.3.2 // indirect
```

Comment:
Why are there so many changes in this file? You do not have to have changes in it.